x/merkledb/db.go

```diff
@@ -997,21 +997,22 @@ func (db *merkleDB) VerifyChangeProof(
 	smallestPath := newPath(start)
 
 	// Make sure the start proof, if given, is well-formed.
-	if err := verifyProofPath(proof.StartProof, smallestPath); err != nil {
+	if err := verifyProofPath(proof.StartProof, Some(smallestPath)); err != nil {
```

Comment:
start is always meaningful since a nil start key behaves identically to no start key